src/sfctl/tests/app_test.py

```diff
@@ -220,7 +220,7 @@ def test_upload_image_store_timeout_long(self):  #pylint: disable=too-many-local
 
         with open(generated_file_path, 'r') as http_recording_file:
             json_str = http_recording_file.read()
-            vcr_recording = decode(json_str)
+            vcr_recording = json.loads(json_str)
```

Comment:
Are these the only two places this is used? If we can guarantee that we only deserialize "safe" content with the jsonpickle package, then we should be okay.

Reply:
Yes, these are the only spots. Checked with @Christina-Kang